examples/align-gl/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2019 Intel Corporation. All Rights Reserved.
+#  minimum required cmake version: 3.1.0
+cmake_minimum_required(VERSION 3.1.0)
+
+project(RealsenseExamplesAlignGl )
+
+if(BUILD_GRAPHICAL_EXAMPLES)
+    add_executable(rs-align-gl rs-align-gl.cpp ../../third-party/imgui/imgui.cpp ../../third-party/imgui/imgui_draw.cpp ../../third-party/imgui/imgui_impl_glfw.cpp)
```

Comment:
Have you tried to use [example-imgui.hpp](https://github.com/IntelRealSense/librealsense/blob/development/examples/example-imgui.hpp)